packages/docs/content/getting-started/developers/examples.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Examples
+order: 16
```

Comment:
What determines whether we count by ones or tens?

Reply:
So I did 10s for "level 2" and 1s for level 3. I'm open to another scheme, but I needed a way to 1) make sure level 3 could be ordered & 2) that level 2 items that had level 3 items could also be ordered and since those types of level 2 items don't have an mdx file associated with them, this is what I came up with